### PR TITLE
Add pytest.ini for custom markers

### PR DESCRIPTION
**Pull request by take-cheeze**
https://docs.pytest.org/en/latest/mark.html#registering-marks
To suppress warnings in pytest

### CODE REVIEW

**Comment by toslunar** [comment]
Chainer has `slow` marker.  [chainer/testing/attr.py](https://github.com/chainer/chainer/blob/master/chainer/testing/attr.py)

**Comment by toslunar** [comment]
It seems `pytest.ini` somehow conflicts with `test_expand_dims_invalid`.  Could you investigate the cause if you are interested in it?

The priority of the PR is not high at the moment, because just migrating to the latest pytest is a more important task.  Nevertheless I appreciate the contribution because Chainer would eventually have it.

- Currently Chainer uses `pytest<4.2.0` (see #6164).  It's possible that pytest needs to be fixed.
- The warning message is from pytest 4.5.0.

**Comment by take-cheeze** [comment]
Test failures seems to be caused by this file so will investigate in spare time:
https://github.com/chainer/chainer/blob/9472d08c4fb431a291405af4bd93726e051240ac/tests/chainerx_tests/unit_tests/routines_tests/test_manipulation.py#L854

**Comment by take-cheeze** [comment]
Seems warning wasn't treated as error somehow with pytest.ini.
Maybe we should switch to: https://github.com/fschulze/pytest-warnings

**Comment by niboshi** [comment]
Perhaps #6653 is related.

File: tests/chainerx_tests/unit_tests/routines_tests/test_manipulation.py [hunk 1]
```diff
@@ -895,6 +896,7 @@ def forward_xp(self, inputs, xp):
     ((1, 1, 2), -4)
 ])
 def test_expand_dims_invalid(xp, shape, axis):
+    warnings.simplefilter('error', DeprecationWarning)
```
Comment (toslunar) [inline tests/chainerx_tests/unit_tests/routines_tests/test_manipulation.py]: Could you create a new pull request for this fix?

Comment (take-cheeze) [reply tests/chainerx_tests/unit_tests/routines_tests/test_manipulation.py]: Created: #7824

**Comment by toslunar** [comment]
> Seems warning wasn't treated as error somehow with pytest.ini.

It seems PyTest cannot find two or more configuration files.
https://github.com/pytest-dev/pytest/blob/6c8bcf601c24759e182fcde5943f1e29270f4d48/src/_pytest/config/findpaths.py#L27-L54

**Comment by take-cheeze** [comment]
Move to #8112

**Pull request closed without merging**